components/transform/transform_local_to_wgs84/transform_local_to_wgs84.py

```diff
@@ -0,0 +1,135 @@
+#!/usr/bin/python
+from asyncio.windows_events import NULL
+import logging
+import os
+import sys
+import json
+import base64
+
+def path_up_to_last(a_last, a_inclusive=True, a_path=os.path.dirname(os.path.realpath(__file__)), a_sep=os.path.sep):
+    return a_path[:a_path.rindex(a_sep + a_last + a_sep) + (len(a_sep)+len(a_last) if a_inclusive else 0)]
+
+components_dir = path_up_to_last("components")
+
+sys.path.append(os.path.join(components_dir, "utils"))
+from imports import *
+
+for imp in ["args", "utils", "get_token", "rdm_pagination_traits", "rdm_list", "transform", "site_detail"]:
+    exec(import_cmd(components_dir, imp))
+
+session = requests.Session()
+
+
+def get_local_grid_to_cartesian_approx_matrix(a_server_url, a_headers, a_site_id, a_transform_rev, a_voxel_extent, a_points):
+   
+    transform_url = "{0}/transform/v1/site/{1}/transform_rev/{2}/enz_to_ecef_approx_matrix".format(a_server_url, a_site_id, a_transform_rev)
+      
+    voxels = []
+    for point in a_points:
+        voxels.append({
+            "i": int(point["e"] / a_voxel_extent),
+            "j": int(point["n"] / a_voxel_extent),
+            "k": int(point["z"] / a_voxel_extent),
+        })
```

Comment:
This will result in duplicate voxels if there is more than one point in a particular voxel